### PR TITLE
Add GET for DCV Delegation UUID endpoint

### LINKED ISSUE #1383: Add GET for /dcv_delegation/uuid
### Current cloudflare-go version

0.76.0

### Description

https://api.cloudflare.com/client/v4/zones/<zone_id>/dcv_delegation/uuid is not currently supported by the Go SDK. 

### Use cases

We need the ability to automate partial zone CNAME DCV delegation after zone creation in order to allow certificates to be auto-renewed.

### Potential cloudflare-go usage

```go
Simple Go SDK support for new endpoint.
```

### References

_No response_

### PR DESCRIPTION
**Pull request by troymjones**
## Description

https://api.cloudflare.com/client/v4/zones/<zone_id>/dcv_delegation/uuid is not currently supported by the Go SDK.
closes #1383 

## Has your change been tested?

Unit tests added in changes

## Types of changes

What sort of change does your code introduce/modify?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.
- [x] This change is using publicly documented in [cloudflare/api-schemas](https://github.com/cloudflare/api-schemas) 
      and relies on stable APIs.

### CODE REVIEW

**Comment by vasilzhigilei** [comment]
Code looks good from SSL team side
Edit: we'll add API docs to unblock this

**Comment by vasilzhigilei** [comment]
API docs are now available: https://developers.cloudflare.com/api/operations/dcv-delegation-uuid-get

File: dcv_delegation.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+// DCVDelegationResponse represents the response from the dcv_delegation/uuid endpoint.
+type DCVDelegationResponse struct {
+	Result DCVDelegation `json:"result"`
+	Response
+	ResultInfo `json:"result_info"`
+}
+
+// GetDCVDelegation gets a zone DCV Delegation UUID
+//
+// TODO: Add API documentation
```
Comment (jacobbednarz) [inline dcv_delegation.go]:
```suggestion
// API documentation: https://developers.cloudflare.com/api/operations/dcv-delegation-uuid-get
```

File: dcv_delegation.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+// DCVDelegationResponse represents the response from the dcv_delegation/uuid endpoint.
+type DCVDelegationResponse struct {
+	Result DCVDelegation `json:"result"`
+	Response
+	ResultInfo `json:"result_info"`
+}
+
+// GetDCVDelegation gets a zone DCV Delegation UUID
```
Comment (jacobbednarz) [inline dcv_delegation.go]:
```suggestion
// GetDCVDelegation gets a zone DCV Delegation UUID.
```

File: dcv_delegation.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+//
+// TODO: Add API documentation
+func (api *API) GetDCVDelegation(ctx context.Context, rc *ResourceContainer) (DCVDelegation, ResultInfo, error) {
```
Comment (jacobbednarz) [inline dcv_delegation.go]:
```suggestion
func (api *API) GetDCVDelegation(ctx context.Context, rc *ResourceContainer, params GetDCVDelegationParams) (DCVDelegation, ResultInfo, error) {
```

File: dcv_delegation.go [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+type DCVDelegation struct {
+	UUID string `json:"uuid"`
+}
+
+// DCVDelegationResponse represents the response from the dcv_delegation/uuid endpoint.
+type DCVDelegationResponse struct {
+	Result DCVDelegation `json:"result"`
+	Response
+	ResultInfo `json:"result_info"`
+}
```
Comment (jacobbednarz) [inline dcv_delegation.go:20]:
```suggestion
}

type GetDCVDelegationParams struct {}
```

File: dcv_delegation_test.go [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+		`, testUuid)
+	}
+
+	mux.HandleFunc("/zones/"+testZoneID+"/dcv_delegation/uuid", handler)
+
+	want := DCVDelegation{
+		UUID: testUuid,
+	}
+
+	actual, _, err := client.GetDCVDelegation(context.Background(), ZoneIdentifier(testZoneID))
```
Comment (jacobbednarz) [inline dcv_delegation_test.go]:
```suggestion
	actual, _, err := client.GetDCVDelegation(context.Background(), ZoneIdentifier(testZoneID), GetDCVDelegationParams{})
```

**Pull request merged**